thalassa/api.py

```diff
@@ -74,7 +74,7 @@ def open_dataset(
     import xarray as xr
 
     default_kwargs: dict[str, T.Any] = dict(
-        mask_and_scale=True,
+        # mask_and_scale=True,
```

Comment:
good question, I forgot to mention this:
`mask_and_scale` is an argument that is not available in the xarray-selafin backend but that seems to exists for all other formats (zarr, grib, netcdf). I don't know how to solve this

Reply:
`mask_and_scale` is about applying the `add_offset` and the `scale_factor`. It's a netcdf thing, but xarray also applies it to zarr. I don't know if it makes sense in selafin, but even if it doesn't, I would add it as a do-nothing argument. Either that or I would add a `**kwargs` so that it can be consumed without throwing an exception

Reply:
you mean directly in `xarray-selafin`, correct?